src/components/JsonViewer/components/HighlightedText.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+
+import type {NoStrictEntityMods} from '@bem-react/classname';
+
+interface Props {
+    className: (mods?: NoStrictEntityMods) => string;
+    text: string;
+    start?: number;
+    length?: number;
+    hasComa?: boolean;
+    markBegin?: boolean;
+}
+
+export default function HighlightedText({className, text, start, length, hasComa}: Props) {
+    const comma = hasComa ? <React.Fragment>,&nbsp;</React.Fragment> : null;
+
+    if (length! > 0 && start! >= 0 && start! < text.length) {
```

Comment:
You can get rid of not-null assertions by checking if `length` and `start` are defined